### PR TITLE
dreport: Add audit.log Data to Dump

### PR DESCRIPTION
**Pull request by zamiseck**
Add /var/log/audit/audit.log to BMC dumps as audit-log.log. This file contains a log of the events that occurred and who initiated them.

Tested:
~/Code/dumps/2/BMCDUMP.SIMP10R.00000002.20220919155900_out/archive$ ls -la audit-log.log 
 -rw-r--r-- 1 zamiseck zamiseck 3993 Sep 19 10:59 audit-log.log

### CODE REVIEW

File: tools/dreport.d/plugins.d/auditlog [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+#
+
+. $DREPORT_INCLUDE/functions
+
+desc="Audit Log"
+file_name="audit-log.log"
+
+if [ -e "/var/log/audit/audit.log" ]; then
+  command="cat /var/log/audit/audit.log"
+  add_cmd_output "$command" "$file_name" "$desc"
```
Comment (spinler) [inline tools/dreport.d/plugins.d/auditlog:14]: You can use the 'add_copy_file' function here to just copy over the file.

**Comment by ojayanth** [comment]
What is the max size of the audit log ( before compression?).  

Also please share upstream reveiw link here.

**Comment by ojayanth** [comment]
upstream reveiw related to customize p10 dump size . https://gerrit.openbmc.org/c/openbmc/openbmc/+/50579

**Comment by zamiseck** [comment]
@ojayanth I have the config file for audit set so the log file size does not grow past 10 MB. This can be changed if there is a more suitable size that should be used.

Here is a link to the audit discussion  upstream: https://gerrit.openbmc.org/c/openbmc/bmcweb/+/55532

**Review by ojayanth — APPROVED** [review]

**Pull request merged**